dev/com.ibm.websphere.appserver.features/visibility/public/mpMetrics-5.0/io.openliberty.mpMetrics-5.0.feature

```diff
@@ -16,9 +16,10 @@ Subsystem-Name: MicroProfile Metrics 5.0
   com.ibm.websphere.appserver.anno-2.0, \
   com.ibm.websphere.appserver.monitor-1.0
 -bundles=com.ibm.ws.microprofile.metrics.common, \
- io.openliberty.microprofile.metrics.internal.3.0.jakarta, \
- io.openliberty.microprofile.metrics.internal.cdi.3.0.jakarta, \
- io.openliberty.microprofile.metrics.internal.private, \
- io.openliberty.microprofile.metrics.internal.public
+ io.openliberty.io.smallrye.common.jakarta, \
+ com.ibm.ws.cdi.interfaces.jakarta, \
```

Comment:
```suggestion
```
Already in cdi-4.0 so not needed to be explicitly listed for this feature as well.